<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BANCUSTOMQA05" ignored="1" name="5:WorkFLows_USMFFundOrderTestCases" tests="12" failures="1" timestamp="29 Mar 2018 06:43:29 GMT" time="81.877" errors="0">
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at utilities.webdriverfunctions.WebDriverMethods.capture_screenshot(WebDriverMethods.java:149)
	at testcases.commontoall.CommonToAllTests.end_test_update_results(CommonToAllTests.java:150)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="@AfterMethod end_test_update_results" time="0.0" classname="testcases.commontoall.CommonToAllTests">
    <skipped/>
  </testcase> <!-- @AfterMethod end_test_update_results -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="74.919" classname="testcases.workflows.USMFWorkFlowTests">
    <failure type="org.openqa.selenium.WebDriverException" message="org.apache.http.conn.HttpHostConnectException: Connect to localhost:32976 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
Build info: version: &amp;apos;3.6.0&amp;apos;, revision: &amp;apos;6fbf3ec767&amp;apos;, time: &amp;apos;2017-09-27T15:28:36.4Z&amp;apos;
System info: host: &amp;apos;BANCUSTOMQA05&amp;apos;, ip: &amp;apos;10.96.85.207&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0_121&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: org.apache.http.conn.HttpHostConnectException: Connect to localhost:32976 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'BANCUSTOMQA05', ip: '10.96.85.207', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_121'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:356)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:458)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:348)
	at pages.pageclasses.OrderMgtPage.expandTheFiltersImage(OrderMgtPage.java:2661)
	at testcases.workflows.USMFWorkFlowTests.USMF_Work_Flow(USMFWorkFlowTests.java:131)
	at testcases.workflows.USMFWorkFlowTests.CITI_Fund_Order_Entry_USMF_NonTP(USMFWorkFlowTests.java:41)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to localhost:32976 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:359)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:138)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:86)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:161)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	... 36 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:75)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	... 49 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_NonTP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_NonTP -->
  <testcase name="CITI_Fund_Transfer_USMF_TP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Transfer_USMF_TP -->
  <testcase name="CITI_Fund_Transfer_USMF_TP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Transfer_USMF_TP -->
  <testcase name="CITI_Fund_Transfer_USMF_TP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Transfer_USMF_TP -->
  <testcase name="CITI_Fund_Transfer_USMF_TP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <skipped/>
  </testcase> <!-- CITI_Fund_Transfer_USMF_TP -->
  <testcase name="CITI_Fund_Order_Entry_USMF_TP" time="0.0" classname="testcases.workflows.USMFWorkFlowTests">
    <ignored/>
  </testcase> <!-- CITI_Fund_Order_Entry_USMF_TP -->
</testsuite> <!-- 5:WorkFLows_USMFFundOrderTestCases -->
